Classes/CatchUpHook/DocumentUriPathProjectionHook.php

```diff
@@ -197,25 +195,21 @@ private function handleNodeWasMoved(NodeAggregateWasMoved $event, \Closure $clos
             return;
         }
 
-        foreach ($event->nodeMoveMappings as $moveMapping) {
-            /* @var \Neos\ContentRepository\Core\Feature\NodeMove\Dto\OriginNodeMoveMapping $moveMapping */
-            foreach ($moveMapping->newLocations as $newLocation) {
-                /* @var $newLocation CoverageNodeMoveMapping */
-                $node = $this->findNodeByIdAndDimensionSpacePointHash($event->nodeAggregateId, $newLocation->coveredDimensionSpacePoint->hash);
-                if ($node === null) {
-                    // node probably no document node, skip
-                    continue;
-                }
-
-                $closure($node, $this->getNodeAddress($event->contentStreamId, $newLocation->coveredDimensionSpacePoint, $node->getNodeAggregateId()), $this->contentRepository->id);
-
-                $descendantsOfNode = $this->getState()->getDescendantsOfNode($node);
-                array_map(fn (DocumentNodeInfo $descendantOfNode) => $closure(
-                    $descendantOfNode,
-                    $this->getNodeAddress($event->contentStreamId, $newLocation->coveredDimensionSpacePoint, $descendantOfNode->getNodeAggregateId()),
-                    $this->contentRepository->id
-                ), iterator_to_array($descendantsOfNode));
+        foreach ($event->succeedingSiblingsForCoverage as $interdimensionalSibling) {
```

Comment:
Migrated in a similar fashion as the DocumentUriPathProjection here:
https://github.com/neos/neos-development-collection/pull/4993/files#diff-2e3df51a3340b51e08cf9d2ba5aade329a8c0e073354c1d5a72e42c3aaf5a127R618